### PR TITLE
Enhance SDK documenation

### PR DESCRIPTION
**Pull request by Bobbins228**
# Issue link

Relates to: [RHOAIENG-9511](https://issues.redhat.com/browse/RHOAIENG-9511)
# What changes have been made

# Verification steps

N/A
## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [x] Testing is not required for this change

### CODE REVIEW

File: docs/ray-cluster-interaction.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* `appwrapper: bool # Default: False` -> When set to `True` allows the user to list queued AppWrappers.
+
+
+
+## list_all_clusters()
+The `list_all_clusters()` function will return a list of detailed descriptions of Ray Clusters to the console by default. It accepts the following parameters:
+* `namespace: str # Required` -> The namespace you want to retrieve the list from.
+* `print_to_console: bool # Default: True` -> A boolean that allows the user to print the list to their console.
+
+<br>
```
Comment (Ygnas) [inline docs/ray-cluster-interaction.md:49]:
```suggestion
<br>

```
Inline html for compatibility should be used at the end of the line, as in this case it breaks the link on the next line unless new line is added after `<br>`

File: docs/ui-widgets.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+<img title="view clusters" alt="An image of the view clusters ui table" src="images/ui-view-clusters.png">
+
+Above is a list of two Ray Clusters `raytest` and `raytest2` each of those headings is clickable and will update the table to view the selected Cluster's information.
+There are three buttons under the table `Cluster Down`, `View Jobs` and `Open Ray Dashboard`.<br>
+* The `Cluster Down` button will delete the selected Cluster.
+* The `View Jobs` button will try to open the Ray Dashboard's Jobs view in a Web Browser. The link will also be printed to the console.
+* The `Open Ray Dashboard` button will try to open the Ray Dashboard view in a Web Browser. The link will also be printed to the console.
+
+The UI Table can be viewed by calling the following function.
+```
```
Comment (Ygnas) [inline docs/ui-widgets.md:26]:
```suggestion
```python
```
Maybe add this for syntax highlighting?

File: docs/ray-cluster-interaction.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Ray Cluster Interaction
+
+The CodeFlare SDK offers multiple ways to interact with Ray Clusters including the below methods.
+
+## get_cluster()
+The `get_cluster()` function is used to initialise a `Cluster` object from a pre-existing Ray Cluster/AppWrapper. <br>
+Below is an example of it's usage:
+```
```
Comment (Ygnas) [inline docs/ray-cluster-interaction.md:8]:
```suggestion
```python
```
Maybe add this for syntax highlighting?

**Comment by Bobbins228** [comment]
Closing in favour of #709

**Pull request closed without merging**